shop/src/app/app.component.ts

```diff
@@ -9,5 +8,5 @@ import { HttpClient } from '@angular/common/http';
 export class AppComponent {
   title: string = 'shop';
 
-  constructor (private http: HttpClient) {}
+  constructor () {}
```

Comment:
You can also delete this constructor if it's empty

Reply:
Erased